src/main/java/bio/terra/service/dataset/flight/ingest/IngestValidateScratchTableFilerefsStep.java

```diff
@@ -42,7 +43,7 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
 
     var tableServiceClient = azureAuthService.getTableServiceClient(storageAuthInfo);
     Table table = IngestUtils.getDatasetTable(context, dataset);
-    var tableName = IngestUtils.getSynapseIngestTableName(context.getFlightId());
+    var scratchTableName = IngestUtils.getSynapseScratchTableName(context.getFlightId());
 
     // For each fileref column, scan the staging table and build an array of file ids
```

Comment:
This comment makes me think that the original intention may have been to validate the file references off of the scratch (staging) table, rather than the published Parquet files.